src/components/HomePage/HothDescription.js

```diff
@@ -84,10 +84,11 @@ export default function HothDescription() {
 					</Grid>
 					<Grid item sm={10} md={6}>
 						<Typography variant='body1' style={{ paddingBottom: isSmall ? theme.spacing(2) : 0 }}>
-							Hack on the Hill (HOTH) 7 is a 12-hour,{' '}
+							Hack off the Hill (HOTH 8) is a {' '}
 							<strong>beginner-friendly</strong>
-							{' '}hackathon designed to give beginners a glimpse into what a real
-							hackathon would be and feel like.
+							{' '}hackathon designed to be your first hackathon experience.
+							{' '}Due to the ongoing global pandemic,
```

Comment:
```suggestion
							Due to the ongoing global pandemic,
```

the explicit space is only needed around a JSX tag.